src/components/Hooks/useFormWithValidation.js

```diff
@@ -0,0 +1,33 @@
+import React, { useCallback } from "react";
+
+export function useFormWithValidation() {
```

Comment:
**Could be improved**

Please, use `useFormWithValidation` to show errors under the inputs.

Reply:
Done